### DISCUSSION TITLE
TGA flipped

### DISCUSSION THREAD

**Discussion by dmdmdm** [ImageMagick/ImageMagick#3576] [Development]
Hi, sorry if this is the wrong place to report a bug.
Using imdisplay my TGA file shows upside down and reversed:
![imdisplay_screenshot](https://user-images.githubusercontent.com/3187057/115784984-db9a3f00-a38c-11eb-94f2-24d9a55b60d7.png)
When I display the same file in LibreOffice Draw its correct:
![libreoffice_draw_screenshot](https://user-images.githubusercontent.com/3187057/115785299-46e41100-a38d-11eb-85a8-c70ff1aaf7cc.png)
I suspect that ImageMagick isn't respecting the origin bit in the TGA header:
![tga_header](https://user-images.githubusercontent.com/3187057/115785697-cbcf2a80-a38d-11eb-8064-cb49049ea33b.png)

The TGA is attached in a zip
[FireSafety01.zip](https://github.com/ImageMagick/ImageMagick/files/6361302/FireSafety01.zip)

This works in older versions of ImageMagick but doesn't work in the current version 7.0.11-8 Windows.
Thanks

**Comment by urban-warrior** [comment]
Add `-auto-orient` to your command-line.  ImageMagick notes the orientation of BottomLeft for your image with `identify -verbose FireSafety01.tga`.  However, ImageMagick does not automatically re-orient the image pixels to TopLeft unless you ask it to with `-auto-orient`.

**Reply by dmdmdm** [reply]
Thanks for the reply and thanks for ImageMagick!  As the screenshot of the format info says, that bit must be 0 (lower left corner) so every TGA has it that way.  In other words its not really an orientation choice the way EXIF is set in a JPG when somebody rotates their phone.  Its more a detail of the image format like big-endian or little-endian numbers.  Perhaps `-auto-orient` should be the default for `imdisplay` since its for showing images to humans?
